backend/battles/migrations/0007_auto_20210113_1745.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.17 on 2021-01-13 17:45
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Battle = apps.get_model("battles", "Battle")
+    BattleTeam = apps.get_model("battles", "BattleTeam")
+
+    db_alias = schema_editor.connection.alias
+
+    for battle_team in BattleTeam.objects.using(db_alias).all():
+        Battle.objects.using(db_alias).filter(
+            id=battle_team.battle.id,
+        ).update(
+            creator_pokemon_1=battle_team.pokemon_1,
+            creator_pokemon_2=battle_team.pokemon_2,
+            creator_pokemon_3=battle_team.pokemon_3,
+        )
+
+
+def reverse_func(apps, schema_editor):
```

Comment:
It's ok to have this here, but usually in this situation we wouldn't write a reverse function

Reply:
Hmm, could you please tell me why?

Reply:
For the context on this particular use case in this application there's no gain in setting this fields as null. you can always go back and forth with this migration and the result is the same. Eg.: suppose there's a bug in `forwards_func`. You could just revert the migration (doing nothing), fix `forwards_func` and then run the migration again. 

You will need to evaluate case by case if you need or not a migration reversion function

Reply:
Got it!